spec/01-unit/27-batch_queue_spec.lua

```diff
@@ -0,0 +1,30 @@
+
+local BatchQueue = require "kong.tools.batch_queue"
```

Comment:
Thank you for getting this started, we should have had this since the beginning.